capstone/src/test/java/com/google/sps/servlets/clubs/EditClubServletTest.java

```diff
@@ -171,7 +174,61 @@ public void doPost_editLabels() throws ServletException, IOException {
     Assert.assertEquals(labels.get(1), "hgbdi-sg");
   }
 
-  private void prepClubEnv() {
+  @Test
+  public void doPost_makeClubExclusive() throws ServletException, IOException {
+    Entity startingEntity = makeClubEntity();
+    startingEntity.setProperty(Constants.EXCLUSIVE_PROP, false);
+    datastore.put(startingEntity);
+    String newDescription = "new test description";
+    String newWebsite = "new-website.com";
+    String newOfficer = "kakm@google.com";
+    when(request.getParameter(Constants.PROPERTY_NAME)).thenReturn(SAMPLE_CLUB_NAME);
+    when(request.getParameter(Constants.OFFICER_PROP)).thenReturn(TEST_EMAIL + "," + newOfficer);
+    when(request.getParameter(Constants.DESCRIP_PROP)).thenReturn(newDescription);
+    when(request.getParameter(Constants.WEBSITE_PROP)).thenReturn(newWebsite);
+    when(request.getParameter(Constants.EXCLUSIVE_PROP)).thenReturn("on");
+    when(request.getUserPrincipal()).thenReturn(principal);
+    when(principal.getName()).thenReturn(TEST_EMAIL);
+    editClubServlet.doPost(request, response);
+
+    Query query =
+        new Query(Constants.CLUB_ENTITY_PROP)
+            .setFilter(
+                new FilterPredicate(
+                    Constants.PROPERTY_NAME, FilterOperator.EQUAL, SAMPLE_CLUB_NAME));
+    Entity clubEntity = datastore.prepare(query).asSingleEntity();
+    Assert.assertNotNull(clubEntity);
+    Assert.assertTrue((Boolean) clubEntity.getProperty(Constants.EXCLUSIVE_PROP));
+  }
+
+  @Test
+  public void doPost_makeClubInclusive() throws ServletException, IOException {
```

Comment:
super nit: name should say notExclusive instead of inclusive only because exclusive is a boolean and not an ENUM with different explicit options like Exclusive, Inclusive, etc.